lib/matplotlib/patches.py

```diff
@@ -920,7 +921,9 @@ class StepPatch(PathPatch):
     """
     A path patch describing a stepwise constant function.
 
-    The path is unclosed. It starts and stops at baseline.
+    By default the path is unclosed and starts and stops at baseline.
+    If an array is passed to *baseline*, while ``fill=False`` a closed
+    shape is drawn.
```

Comment:
```suggestion
    If *fill* is True or an array is passed to *baseline*, a closed
    path is drawn.
```

Reply:
Should this be the wording in both the patch and `stairs`? I would say not. It makes some sense to me for the artists because it's lower level and technically that's what's happening, but in plotting-wise if fill is True the outline (path) is not drawn.

Reply:
I agree that the scope is different and thus the wording can be.